control.lua

```diff
@@ -49,9 +48,6 @@ function enable_modifiers(player)
 	if player.character_inventory_slots_bonus ~= 60 then
 		player.character_inventory_slots_bonus  = 60
 	end
-	if player.quickbar_count_bonus ~= 1 then
```

Comment:
Looks like this feature won't be needed anymore since the tool belt is changed completely
agree on that change